.github/workflows/release.yml

```
@@ -0,0 +1,80 @@
+name: Publish package to npm
```

```suggestion
name: Publish package on npm
```

not very critical... but isnt it `publish smth on`?